applications/main/nfc/scenes/nfc_scene_mf_classic_dict_attack.c

```diff
@@ -1,11 +1,16 @@
 #include "../nfc_app_i.h"
 
+#include <bit_lib/bit_lib.h>
 #include <dolphin/dolphin.h>
 #include <lib/nfc/protocols/mf_classic/mf_classic_poller.h>
 
 #define TAG "NfcMfClassicDictAttack"
 
+// TODO: Fix lag when leaving the dictionary attack view after Hardnested
+// TODO: Re-enters backdoor detection between user and system dictionary if no backdoor is found
+
```

Comment:
What about these TODOs? Are they still actual or not?

Reply:
They are actual, but not for this PR necessarily. I have them documenting the planned performance and code style improvements. Ideally with a follow-up PR, so users have a working process before I've had the chance to fully optimize it.

Reply:
More context: to the best of my knowledge the PR is stable and testing is appreciated. The cost of the performance TODOs will add up to milliseconds for the vast majority of attacks.

For the two TODOs referenced here, I've observed under certain conditions Hardnested taking a few seconds to return from the dictionary attack. The backdoor re-entry cost is somewhere in the order of milliseconds.

Reply:
Can we change TODO to something else, otherwise it will trigger our TODO checker?